pkg/aws/snow_test.go

```diff
@@ -0,0 +1,43 @@
+package aws
```

Comment:
can we test this from `aws_test`? or do the tests depend on something unexported?